website/src/docs/hotchocolate/filtering/bool-filter.md

```diff
@@ -0,0 +1,57 @@
+### Boolean Filter
```

Comment:
Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.